docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2138,3 +2138,54 @@ Result:
 
 -   [tcp_port](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-tcp_port)
 
+## currentProfiles {#current-profiles}
+
+Returns a list of the current [settings profiles](../../operations/access-rights.md#settings-profiles-management) for the current user. 
+
+The settings profile can be specified at the user's definition (see [CREATE USER](../../sql-reference/statements/create/user.md#create-user-statement), `PROFILE` clause) and can be changed with the [SET profile](../../sql-reference/statements/set.md#query-set) statements lately. 
```

Comment:
The part about `SET PROFILE` should be written first, because it's more important for the explanation.
`SET PROFILE` is a command, it should be written uppercase.